tools/doc/versions.js

```diff
@@ -31,6 +31,13 @@ const getUrl = (url) => {
   });
 };
 
+const createMaster = (masterPath, file) => {
```

Comment:
this function only is used once, why not code inline?

Reply:
You are right. My bad.
Thanks for the suggestion.